addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -3500,6 +3500,16 @@ public Action:TF2Items_OnGiveNamedItem(client, String:classname[], iItemDefiniti
 
 	switch(iItemDefinitionIndex)
 	{
+		case 237, 265: // Rocket & Sticky Jumper
```

Comment:
Personally I think this will be confusing because of the distinctive color and "no damage" that's associated with it.